grakn-graph/src/test/java/ai/grakn/graph/internal/EntityTypeTest.java

```diff
@@ -592,27 +594,21 @@ public void checkSubTypeCachingUpdatedCorrectlyWhenChangingSuperTypes(){
         EntityType e6 = graknGraph.putEntityType("entityType6").superType(e5);
 
         assertEquals(4, e1.subTypes().size());
-        assertTrue("e1 subtypes does not contain e1", e1.subTypes().contains(e1));
-        assertTrue("e1 subtypes does not contain e2", e1.subTypes().contains(e2));
-        assertTrue("e1 subtypes does not contain e3", e1.subTypes().contains(e3));
-        assertTrue("e1 subtypes does not contain e4", e1.subTypes().contains(e4));
+        assertThat(e1.subTypes(), containsInAnyOrder(e1, e2, e3, e4));
```

Comment:
Yay! I don't care if you do this or not, but I thought you should know `containsInAnyOrder` will also confirm the size is correct.